clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -432,4 +432,19 @@ DPCT_NON_ENUM_OPTION(
         "Query functionally compatible SYCL API to migrate CUDA API."),
     llvm::cl::value_desc("api"), llvm::cl::cat(DPCTCat), llvm::cl::Optional)
 
+DPCT_ENUM_OPTION(
+    DPCT_OPT_TYPE(static llvm::cl::bits<HelperFuncPreference>), Preferences,
+    "helper-function-preference",
+    DPCT_OPTION_VALUES(DPCT_OPT_ENUM(
+        "no-queue-device", int(HelperFuncPreference::NoQueueDevice),
+        "Call SYCL API to get queue and device instead of calling helper function. Notice\n"
+        "that the runtime behavior may not be the same with the original code because\n"
+        "there is only one global definition of SYCL queue and device, which will be\n"
+        "passed to place where SYCL queue and device are expected.\n",
+        false)),
+    llvm::cl::desc("The preference of helper function usage in migration.\n"
+                   "The values are:\n"),
```

Comment:
please add test case for:  
void foo(){
cudaStream_t  s; 
}

Reply:
added